modules/andes-core/broker/src/main/java/org/wso2/andes/store/cassandra/CQLBasedAndesContextStoreImpl.java

```diff
@@ -748,6 +748,13 @@ public void deleteMessageIdsByQueueName(String queueName) throws AndesException
     /**
      * {@inheritDoc}
      */
+    @Override public void clearSlotStorage() throws AndesException {
```

Comment:
Override annotation should be on a separate line. Please do the same thing in other places as well.